var/spack/repos/builtin/packages/wrf/patches/4.2/add_aarch64.patch

```diff
@@ -35,32 +35,29 @@ index 7767a81e..bfd2a802 100644
 +PROMOTION       =       #-fdefault-real-8
 +ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM
 +CFLAGS_LOCAL    =       -w -O3 -c -march=native -mtune=native
-+DFLAGS_LOCAL   =       
-+CPLUSPLUSLIB    =       
++DFLAGS_LOCAL   =
++CPLUSPLUSLIB    =
 +ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
-+FCOPTIM         =       -O3 -ftree-vectorize -funroll-loops -march=native -mtune=native
-+FCREDUCEDOPT	=       $(FCOPTIM)
-+FCNOOPT		=       -O0
++FCOPTIM         =       -Ofast -fno-expensive-optimizations -fno-reciprocal-math -fsigned-zeros -fno-unsafe-math-optimizations -funroll-loops -march=native -mtune=native
```

Comment:
These shouldn't be part of the optimizations used by the build system in Spack:
```suggestion
+FCOPTIM         =       -Ofast -fno-expensive-optimizations -fno-reciprocal-math -fsigned-zeros -fno-unsafe-math-optimizations -funroll-loops
```
If we use `-march=native` we might render the `target=...` on nodes meaningless.

Reply:
Done. Removed `-march` and `-mcpu`.